bin/package_build.py

```diff
@@ -15,45 +15,10 @@ def purify(dirty):
 	cleaner = clean.replace('"', '')
 	return cleaner
 
-def debian():
-	global DATA, DATA_FILE_LOCATION
-	q = ['Debian_Buster_List.json', 'Debian_Bullseye_List.json', 'Debian_Bookworm_List.json']
-	urls = ['http://ftp.debian.org/debian/dists/buster/main/binary-s390x/Packages.gz', 'http://ftp.debian.org/debian/dists/bullseye/main/binary-s390x/Packages.gz', 'http://ftp.debian.org/debian/dists/bookworm/main/binary-s390x/Packages.gz']
```

Comment:
So you'll want to remove 'Debian_Buster_List.json', & the url that contains "buster"